project/settings.sc

```diff
@@ -365,7 +368,7 @@ trait CliLaunchers extends SbtModule { self =>
   }
 
   def nativeImage =
-    if (Properties.isLinux && arch == "x86_64")
+    if (Properties.isLinux && arch == "x86_64" && isCI)
```

Comment:
Why we want to build `linux-docker-image` only in `CI`?

Reply:
This is oposite - we do not want to use docker on local development since it takes ages to build the project